Commands/MessageCount/countRewards.go

```diff
@@ -0,0 +1,28 @@
+{{/*
+    Trigger Type: Regex
+    Trigger     : .*
+*/}}
+
+{{$rewards := dict 50 ROLE-ID 75 ROLE-ID 100 ROLE-ID 125 ROLE-ID 150 ROLE-ID}}
+{{/* The rewards count goes from 50 messages and upwards, you can change/add to this if you want */}}
+
+{{if ($db := dbGet .User.ID "msgCount")}}
+{{/* Checks if they have a message count */}}
+
+    {{range $k, $v := $rewards -}}
+    {{/* Ranges the KEY:VALUE pairs of the rewards */}}
```

Comment:
```suggestion
    {{- /* Ranges the KEY:VALUE pairs of the rewards */}}
```
comment blocks need to be stripped too